frontend/src/components/ContentPanel/ContentPanelHeader/index.tsx

```diff
@@ -28,8 +30,11 @@ export const ContentPanelHeader: React.FC<ContentPanelHeaderProps> = ({
                         setCategory={setCategory}
                     />
                 </div>
-                <div className="flex-1 z-10 pl-8">
-                    <DropdownMenu addPromptBoxes={addPromptBoxes} />
+                <div className="flex-1 z-30 pl-8">
```

Comment:
Putting this at z-30 causes the dropdown to be rendered over the navigation bar.
Consider switching this to `z-[15]`

![image](https://user-images.githubusercontent.com/105944267/215729508-03873b24-6902-4902-a21f-8f8e9317fd59.png)

Reply:
Good catch!